crates/ere-dockerized/src/input.rs

```diff
@@ -0,0 +1,64 @@
+use crate::{ErezkVM, error::CommonError};
+use serde::Serialize;
+use zkvm_interface::{Input, InputItem};
+
+impl ErezkVM {
+    pub fn serialize_object(
+        &self,
+        obj: &(impl Serialize + ?Sized),
+    ) -> Result<Vec<u8>, CommonError> {
+        match self {
+            // Issue for tracking: https://github.com/eth-act/ere/issues/4.
+            Self::Jolt => unimplemented!(),
+            // Issue for tracking: https://github.com/eth-act/ere/issues/63.
+            Self::Nexus => unimplemented!(),
+            // FIXME: Instead of using `openvm::serde::to_vec`, we use Risc0's
+            //        serializer, because OpenVM uses the same one, to avoid the
+            //        duplicated extern symbol they export.
+            //        It'd be better to have each zkvm provides their
+            //        lightweight serde crate.
```

Comment:
Worth also creating an issue?

Reply:
>   It'd be better to have each zkvm provides their lightweight serde crate.

This seems like a good suggestion, can we create an issue so we remember to ask them in the future to do this?